### DISCUSSION TITLE
SQS back off time equivalent in 3.x.x

### DISCUSSION THREAD

**Discussion by timothysugar** [awspring/spring-cloud-aws#713] [Q&A]
While using Starter SQS from 3.0.0-RC1, at times my queue consumer suffers from network connectivity failures with AWS. I see a very large number of errors raised due to reconnection attempts. My immediate issue is that this floods the logs with errors. It appears to me that in the 2.x versions, this behaviour could be affected using the `cloud.aws.sqs.listener.back-off-time` configuration property, but I don't see any equivalent for 3.x.

Is that correct? Is there any suggested way of affecting this behaviour?

Thank you for your contributions to this project.

**Comment by tomazfernandes** [comment]
Hey @timothysugar, thanks for bringing this up.

Yeah, we don't have any such mechanism for 3.0. I'll convert this to an issue and we can discuss there.
